x-pack/plugins/infra/server/features.ts

```diff
@@ -24,49 +24,69 @@ export const METRICS_FEATURE = {
   management: {
     insightsAndAlerting: ['triggersActions'],
   },
-  alerting: [METRIC_THRESHOLD_ALERT_TYPE_ID, METRIC_INVENTORY_THRESHOLD_ALERT_TYPE_ID],
+  alerting: [
+    METRIC_THRESHOLD_ALERT_TYPE_ID,
+    METRIC_INVENTORY_THRESHOLD_ALERT_TYPE_ID,
+    LOG_DOCUMENT_COUNT_ALERT_TYPE_ID,
+  ],
   privileges: {
     all: {
       app: ['infra', 'metrics', 'kibana'],
       catalogue: ['infraops', 'metrics'],
-      api: ['infra'],
+      api: ['infra', 'rac'],
```

Comment:
Do I need `rac` here? I found it in the APM plugin and it made total sense to me to add it here as well. However, I would expect that adding this would authorize the infra plugin to access the api route `/internal/rac/alerts/bulk_update`, but it didn't. I still kept getting Forbidden. What is the api section used for and what do we specify here?